frontend/src/v5/ui/themes/theme.ts

```diff
@@ -432,7 +432,13 @@ export const theme = createTheme({
 					color: null,
 				},
 				colorPrimary: {
-					color: COLOR.PRIMARY_MAIN,
+					color: COLOR.BASE_LIGHT,
```

Comment:
What is this for? (just checking this doesn't impact other components )

Reply:
![image](https://user-images.githubusercontent.com/62481250/207575557-827d39ba-0f58-4c04-b4f5-0a10c1749771.png)
It's for these radio controls. (Can also be seen in Project Permissions)